tests/structures/test_generator.py

```diff
@@ -71,28 +71,158 @@ def fizz_buzz(self):
             for i in myinterview.fizz_buzz():
                 print(i)
             """)
-    
+
     def test_yield_from_not_used(self):
         """Yield from is currently not implemented.
-        Unused yield from statements must not break 
+        Unused yield from statements must not break
         the program compilation, but only ensue a warning."""
         self.assertCodeExecution("""
-            
             def unused():
                 yield from range(5)
 
-            print('Hello, world!')
+                print('Hello, world!')
```

Comment:
The (lack of) indentation on this line was intentional. The test needs *some* output to verify that *something* ran. The dummy "Hello world" is proof that the code ran. If it's in the `unused()` method, but `unused()` isn't invoked, then the print will never happen. 

This entire test `test_yield_from_not_used` case will also become redundant as soon as `yield from` exists; the purpose of this test is to validate that just having `yield from` in your code won't cause a problem, as long as the yield from isn't actually on a live code path.